addons/nuxeo-binary-metadata/src/main/java/org/nuxeo/binary/metadata/internals/MetadataMappingDescriptor.java

```diff
@@ -45,6 +46,10 @@
 
     @XNode("@ignorePrefix")
     protected Boolean ignorePrefix = true;
+    
+    /** @since 11.1 */
+    @XNode("@readOnly")
+    protected Boolean readOnly = false;
```

Comment:
This must be just `protected Boolean readOnly;`

Reply:
👍